lib/gui/models/store.js

```diff
@@ -336,21 +336,6 @@ const storeReducer = (state = DEFAULT_STATE, action) => {
         });
       }
 
-      if (!action.data.extension) {
-        throw errors.createError({
-          title: 'Missing image extension'
```

Comment:
There are a few in the unit tests (which is why CI is failing) – just wanted to get this out quick, because I felt I was missing another, nicer way to this, as you pointed out above.